src/adapters/github.js

```diff
@@ -118,12 +118,17 @@ class GitHub extends PjaxAdapter {
     const pullNumber = isPR && showOnlyChangedInPR ? typeId : null
 
     // Skip non-code page unless showInNonCodePage is true
-    if (!showInNonCodePage && type && !~['tree', 'blob'].indexOf(type)) {
+    if (!showInNonCodePage && type && !~['tree', 'blob', 'commit'].indexOf(type)) {
```

Comment:
Hi @stevennoto, the PR looks nice, just one minor improvement. How about introduce a variable for `['tree', 'blob', 'commit'].indexOf(type) >= 0` to represent for "current page is code page" ?

Reply:
Great idea! I'll fix this tonight.